lesson_25/lj-template/src/pages/Home/program-submission/App.tsx

```diff
@@ -0,0 +1,19 @@
+import ProgramSubmission from './ProgramSubmission';
+import {Route, BrowserRouter as Router, Routes} from 'react-router-dom';
+
+import Home from '../Home';
+
+function App() {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route
+          path="http://localhost:5173/program-submission"
```

Comment:
You actually want to add changes to the router configured at `main.tsx` in the `src/` folder..